README.md

```diff
@@ -39,13 +39,34 @@ require('lualine').setup({
 
 ### Overrides examples
 
-#### Replace `IncSearch` group with foreground set to `#FFFFFF`
+#### Transparency
 
 ```lua
+local user_ayu = {}
+
+--- Example visual select colors
+--  make sure you set vim.o.backgorund='light' in your init.lua
+function user_ayu.get_visual_bg()
+    if vim.o.background == 'dark' then
+        return { bg = '#333333' }
```

Comment:
Could you please remove visual `bg` change since it unrelated to the transparency? We have a similar example below.

Reply:
okay